src/components/Disputes/actions/DisputeVoting.js

```diff
@@ -102,8 +114,56 @@ function VotingActions({ canJurorVote, onRequestCommit }) {
   )
 }
 
-const VotingButton = styled(Button)`
-  width: calc((100% - ${2 * GU}px) / 3);
+const RefuseToVoteHint = ({ compactMode, width }) => {
+  const theme = useTheme()
+
+  const Container = compactMode ? Label : 'div'
+
+  return (
+    <div
+      css={`
+        display: flex;
+        align-items: center;
+        justify-content: ${compactMode ? 'center' : 'flex-end'};
+        margin-bottom: ${1.5 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          width: ${width};
+          color: ${theme.help};
+        `}
+      >
+        <Container
+          style={{
```

Comment:
Does `Label` not support the `classNames` prop to allow us to use the `css` prop?

Reply:
When using the `css` prop directly i get a weird `Container is not defined` error

Reply:
Interesting... I wonder if this is because we're using `div` as a string.

Something we could do is use the [`as` prop in `styled-components`](https://styled-components.com/docs/api#as-polymorphic-prop) to swap between the rendered element. In the compact form, we may as well keep the `cursor: pointer` style.

Reply:
Nice